### ISSUE TITLE
Issue when launch Zttlr from a bat script

### ISSUE THREAD

**Issue by mouarfff** [Zettlr/Zettlr#1124]
I launch somes app at the same time with a .bat but Zettlr don't close it after launch and stay open the window with htat : 

Creating placeholder for dir D:\markdown_folder
567 ms: Loaded directory E:\markdown_folder
Starting chokidar ...

(i can see the strange placeholder too ... ?)

I tryed with simply launch it and with start command : 

start /B /D "C:\Program Files\Zettlr" Zettlr.exe

**Comment by canpolat** [comment]
Do you mean: Zettlr does not close the cmd window when you close Zettlr?

**Comment by nathanlesage** [comment]
Yep, I also think this has to do with the batch script — if the issue persists, feel free to comment for re-opening and review!

**Comment by mouarfff** [comment]
Sry but i think it's better to close if you are sure the issue is solved before...

Nothing change.

I re explain what i try to do :  i created a bat file wich launch a lot of app i need for my activities.
It's called "launcher", and when i launch it, the cmd windows open, launch my apps and automaticaly close.

But, since last versions of zettlr, i can't do this, zetlrr stop the .bat and wait for closing.

I need to put this line at the end, and now i have always the cmd windows openned. If i close the window, zetllr close too.
In the cmd window i see what i discribed in my first post

**Comment by canpolat** [comment]
@mouarfff I'm still not sure I understand what you mean, but here is what I did:

I created a `zettlr.bat` file with the following content (just one line):

```
start "" /D "C:\Program Files\Zettlr" /B Zettlr.exe
```

Then I double clicked on it and Zettlr started without any problems. The CMD window closed itself automatically.

I also tried running the batch file via terminal. And I got the same result. Below is the screenshot of the terminal:

![image](https://user-images.githubusercontent.com/459891/88824534-c1994a80-d1c6-11ea-8fa6-30331e4bb05c.png)

In short, I was not able to reproduce the problem you are experiencing. Unless you can provide the step by step reproduction steps for this problem, I don't see how it can be related to Zettlr. You may try to retrace my steps above (Create a new batch file that only starts Zettlr with the line above and try to launch the batch script.)

**Comment by mouarfff** [comment]
`C:\Users\Orion\Desktop>start "" /D "C:\Program Files\Zettlr" /B Zettlr.exe

C:\Users\Orion\Desktop>
Creating placeholder for dir D:\mydocs\markdown
192 ms: Loaded directory E:\mydocs\markdown
Starting chokidar ...`
and the cmd window don't close

If i launch the .bat with double clic, the cmd window don't close.

I use now another app to launch .bat and prevent this behavior (hidden start), but before, zettlr have no issue with that (1 month before).

(i have another bug with the loaded dir D because i changed it to E ? but it's not the issue now.)
